src/components/imagePreviewer/index.jsx

```
@@ -0,0 +1,201 @@
+import Modal from "../modal/Modal";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  addCommentOnVideo,
+  getVideoAllComments,
+  getVideoAllLikes,
+  likeUnlikeVideo,
+} from "../../app/features/videomania";
+import AppInput from "../form/AppInput";
+import ProfileCard from "../card/ProfileCard";
+
+const ImagePreviewer = ({ image, isOpen, onClose, isTop = false }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)